paris-cardview/src/main/java/com/airbnb/paris/cardview/CardViewProxy.kt

```diff
@@ -0,0 +1,33 @@
+package com.airbnb.paris.cardview
+
+import android.content.res.ColorStateList
+import androidx.annotation.Px
+import androidx.cardview.widget.CardView
+import com.airbnb.paris.R2
+import com.airbnb.paris.annotations.Attr
+import com.airbnb.paris.annotations.Styleable
+import com.airbnb.paris.proxies.BaseProxy
+
+@Styleable("Paris_CardView")
+class CardViewProxy(view: CardView) : BaseProxy<CardViewProxy, CardView>(view) {
+
+    @Attr(R2.styleable.Paris_CardView_cardCornerRadius)
+    fun setRadius(@Px radius: Float) {
+        view.radius = radius
+    }
+
+    @Attr(R2.styleable.Paris_CardView_cardBackgroundColor)
+    fun setBackgroundColor(colorStateList: ColorStateList?) {
+        view.setCardBackgroundColor(colorStateList)
+    }
+
+    @Attr(R2.styleable.Paris_CardView_cardElevation)
+    fun setElevation(@Px elevation: Float) {
```

Comment:
Maybe `setCardElevation`? `CardView` extends `View` which already has a `setElevation`. I'm not sure if the implementations are different but best to mirror the property you're mutating IMO.